pkg/client/client.go

```diff
@@ -285,12 +286,13 @@ func get(proxyurl, bucket string, keyname string, wg *sync.WaitGroup, errch chan
 
 	v := hdhashtype == dfc.ChecksumXXHash
 	len, hash, err := readResponse(resp, w, err, fmt.Sprintf("GET (object %s from bucket %s)", keyname, bucket), v)
-	if err != nil && v {
+	if v {
```

Comment:
good catch, hopefully it doesn't triggers new test errors that were hidden before